File: src/main/java/bu/senko/gaykevich/quantumservice/QuantumServiceApplication.java
package bu.senko.gaykevich.quantumservice;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;

@EnableEurekaClient
@SpringBootApplication
public class QuantumServiceApplication {

    public static void main(String[] args) {
        SpringApplication.run(QuantumServiceApplication.class, args);
    }

}

File: src/main/kotlin/by/senko/gaykevich/quantumservice/QuantumServiceApplication.kt
package by.senko.gaykevich.quantumservice

import org.springframework.cloud.netflix.eureka.EnableEurekaClient
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@EnableEurekaClient
@SpringBootApplication
class QuantumServiceApplication

fun main() {
    runApplication<QuantumServiceApplication>()
}
